### DISCUSSION TITLE
Slack Link Broken

### DISCUSSION THREAD

**Discussion by hittecd** [rdbreak/rhcsa8env#54] [Q&A]
The slack link you posted is no longer active. Could you post a new one?

Thanks for all your hard work by the way.

**Comment by rdbreak** [comment]
Sorry for the delay!

This is one irritating feature of Slack invite links: Their never expiring links seem to expire  ¯\_(ツ)_/¯ 

Let's see how long [this "Nonexpiring Link"](https://join.slack.com/t/redhat-certs/shared_invite/zt-qgnlqcck-_0Od17IO5xlkV5BXWaEEEg) last.
